src/Components/Publishing/Sections/FullscreenViewer/ArtworkCaption.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import { Box, BoxProps, Button } from "@artsy/palette"
+import { ArtworkCaption as BaseArtworkCaption } from "../ArtworkCaption"
+
+interface ArtworkCaptionProps extends BoxProps {
+  section: unknown & { type: "artwork"; slug: string }
+}
+
+export const ArtworkCaption: React.FC<ArtworkCaptionProps> = ({
+  section,
+  ...rest
+}) => {
+  return (
+    <Box {...rest}>
+      <BaseArtworkCaption
+        flex="1"
+        artwork={section}
+        isFullscreenCaption
+        linked
+        color="black100"
+      />
+
+      <Button
+        variant="secondaryGray"
+        // TODO: event should be the correct type in palette
+        onClick={(event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+          // TODO: This isn't a real link and that's annoying
+          // <Button> doesn't support `as`, once it does, replace
+          // click handler this with: as={RouterLink} to={`/artwork/${section.slug}`}
```

Comment:
I'd like to update `<Button>` to support this, and I will, but it's a bit of a project for the time being.